src/com/spikes2212/genericsubsystems/drivetrains/HolonomicDrivetrain.java

```diff
@@ -32,36 +41,32 @@ public HolonomicDrivetrain(Consumer<Double> controlLeft, Consumer<Double> contro
 	}
 
 	/**
-	 * Moves the the drivetrain on the x axis
+	 * Moves the X axis of this drivetrain with a given speed.
 	 *

```

Comment:
it doesn't move the access of the drivetrain, it moves the drivetrain on this axis.